Makefile

```diff
@@ -45,10 +45,10 @@ FAILPOINT_ENABLE  := $$(find $$PWD/ -type d | grep -vE "(\.git|tools)" | xargs t
 FAILPOINT_DISABLE := $$(find $$PWD/ -type d | grep -vE "(\.git|tools)" | xargs tools/bin/failpoint-ctl disable)
```

Comment:
Is it better to add a check for the TIDB_EDITION in Makefile, only allow `Community` or `Enterprise` for now? (Since we are now relying on the string value to perform logic instead of simply output, so that a typo makes big difference)